### ISSUE TITLE
PHPStorm 2019.3 no suggestions when working with models (rules)

### ISSUE THREAD

**Issue by techork** [nvlad/yii2support#238]
### What steps will reproduce the problem?
Create a model then create rules
 [['type'], 'required', 'exce' => ,
Or type any attribute name,[['type'], 'required', 'exce' => ,
### What is the expected result?
should give suggestions for except. In 2019.2 was ok

### What do you get instead?
No suggestions for attribute names nor for constraints etc....

### Additional info

| Q                | A
| ---------------- | --------
| IDE Name         | PhpStorm
| IDE Version      | 2019.3
| Plugin version   | 0.10.57
| Yii App Template | basic / advanced

**Comment by techork** [comment]
any updates for this issue?

**Comment by vsalshin** [comment]
This is error occurs when writing rules. 

```
2019-12-09 16:58:13,235 [ 229361]  ERROR - .intellij.psi.PsiReferenceBase - Cannot find manipulator for ArrayAccessExpressionImpl: ''\n            ['id' in com.nvlad.yii2support.objectfactory.ObjectFactoryReference(ArrayAccessExpressionImpl: ''\n            ['id':null) class class com.nvlad.yii2support.objectfactory.ObjectFactoryReference 
java.lang.Throwable: Cannot find manipulator for ArrayAccessExpressionImpl: ''\n            ['id' in com.nvlad.yii2support.objectfactory.ObjectFactoryReference(ArrayAccessExpressionImpl: ''\n            ['id':null) class class com.nvlad.yii2support.objectfactory.ObjectFactoryReference
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:145)
	at com.intellij.psi.PsiReferenceBase.getManipulator(PsiReferenceBase.java:146)
	at com.intellij.psi.PsiReferenceBase.calculateDefaultRangeInElement(PsiReferenceBase.java:109)
	at com.intellij.psi.PsiReferenceBase.getRangeInElement(PsiReferenceBase.java:103)
	at com.intellij.psi.ReferenceRange.getRanges(ReferenceRange.java:24)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.addReferences(SharedPsiElementImplUtil.java:66)
	at com.intellij.psi.impl.SharedPsiElementImplUtil.findReferenceAt(SharedPsiElementImplUtil.java:33)
	at com.intellij.psi.MultiplePsiFilesPerDocumentFileViewProvider.findReferenceAt(MultiplePsiFilesPerDocumentFileViewProvider.java:183)
	at com.intellij.psi.impl.source.PsiFileImpl.findReferenceAt(PsiFileImpl.java:506)
	at com.intellij.codeInsight.TargetElementUtil.findReference(TargetElementUtil.java:117)
	at com.jetbrains.php.completion.PhpCompletionContributor.beforeCompletion(PhpCompletionContributor.java:384)
	at com.intellij.codeInsight.completion.CompletionInitializationUtil.runContributorsBeforeCompletion(CompletionInitializationUtil.java:79)
	at com.intellij.codeInsight.completion.CompletionInitializationUtil.lambda$createCompletionInitializationContext$0(CompletionInitializationUtil.java:53)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteActionWithClass(ApplicationImpl.java:873)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:899)
	at com.intellij.openapi.application.WriteAction.compute(WriteAction.java:110)
	at com.intellij.codeInsight.completion.CompletionInitializationUtil.createCompletionInitializationContext(CompletionInitializationUtil.java:43)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$null$0(CodeCompletionHandlerBase.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$3(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.withTimeout(ProgressIndicatorUtils.java:277)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.withTimeout(CodeCompletionHandlerBase.java:722)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$invokeCompletion$1(CodeCompletionHandlerBase.java:162)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.runUndoTransparentAction(CoreCommandProcessor.java:377)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.invokeCompletion(CodeCompletionHandlerBase.java:175)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.invokeCompletion(CodeCompletionHandlerBase.java:117)
	at com.intellij.codeInsight.completion.CompletionPhase$CommittingDocuments.lambda$scheduleAsyncCompletion$2(CompletionPhase.java:140)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$safeTransferToEdt$8(NonBlockingReadActionImpl.java:340)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:309)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:441)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:906)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:779)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:698)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:421)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by atomrus1993** [comment]
Workaround, install the previous version **2018.3.6** [https://confluence.jetbrains.com/display/PhpStorm/Previous+PhpStorm+Releases](https://confluence.jetbrains.com/display/PhpStorm/Previous+PhpStorm+Releases)

**We are waiting for support 2019.3))**

**Comment by nvlad** [comment]
Hi!
Released plugin version 0.10.57.33.
Check for reproduction.

**Comment by techork** [comment]
@nvlad Thank you, tested on 2020.2.3, everything works fine
